src/game.cpp

```diff
@@ -4622,9 +4622,9 @@ void Game::startDecay(Item* item)
 
 void Game::internalDecayItem(Item* item)
 {
-	const ItemType& it = Item::items[item->getID()];
-	if (it.decayTo != 0) {
-		Item* newItem = transformItem(item, item->getDecayTo());
+	const int32_t decayTo = item->getDecayTo();
+	if (decayTo > 0) {
+		Item* newItem = transformItem(item, decayTo);
 		startDecay(newItem);
```

Comment:
if we arent checking anything there anyway, then do `startDecay(transformItem(item, decayTo));`?